lms/static/js/student_account/views/FormView.js

```diff
@@ -201,24 +209,49 @@
                 renderErrors: function(title, errorMessages) {
                     this.clearFormErrors();
 
-                    this.renderFormFeedback(this.formErrorsTpl, {
-                        jsHook: this.formErrorsJsHook,
-                        title: title,
-                        messagesHtml: HtmlUtils.HTML(errorMessages.join(''))
-                    });
+                    if (title || errorMessages.length) {
+                        this.renderFormFeedback(this.formErrorsTpl, {
+                            jsHook: this.formErrorsJsHook,
+                            title: title,
+                            messagesHtml: HtmlUtils.HTML(errorMessages.join(''))
+                        });
+                    }
                 },
 
                 renderFormFeedback: function(template, context) {
                     var tpl = HtmlUtils.template(template);
                     HtmlUtils.prepend(this.$formFeedback, tpl(context));
+                },
+
+                doOnErrorList: function(id, action) {
+                    var i;
```

Comment:
I _think_ it may have been an (awkward) linting issue. That's why past devs were declaring it earlier in the code. But I think it may not apply here -- will test it.